kcp/Dockerfile

```diff
@@ -24,7 +23,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/manager .
-COPY --from=builder /workspace/chart ./webhook-chart/
 USER 65532:65532
+COPY ../skr/chart/skr-webhook/ skr-webhook/
```

Comment:
I am not sure if this works. AFAIK, you cannot access parent folders from the docker context.
Please have a look at the failing build pipeline.

Reply:
right, I added the copy and remove makefile commands without disturbing the Dockerfile, which is then consistent with other operators